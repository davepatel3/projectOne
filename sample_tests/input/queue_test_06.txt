float
9
2
2
ps
ef 21.3451
ef -56.4068
pb
df
df
gc
eb 12.1768
gc
gf
pf
ef 85.8031
gc
db
ef 83.4507
gs
ps
df
ef -68.0847
ef 67.4387
ef 14.9076
ps
gb
pf
ef 90.3625
gs
db
eb -43.4636
gc
pf
eb 4.3689
pb
gs
ps
pb
eb -26.8484
ef -78.6434
pf
ps
pb
db
gc
df
ps
gc
df
pf
eb 18.623
gc
pf
db
gf
eb -40.9117
gs
db
db
db
db
pf
eb -95.4351
ps
ef -54.2287
ps
ps
db
eb 69.9056
eb 28.3633
df
eb 15.0547
db
ef -66.1259
gb
pb
df
eb -4.79138
eb -46.7355
eb -48.3287
gs
ef -66.3331
eb -58.1758
gc
ef 34.1651
pf
eb -1.4805
qt
