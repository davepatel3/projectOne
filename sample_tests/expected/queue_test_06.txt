[-][-][-][-][-][-][-][-][-]
-56.4068
cyclic_double_queue is empty!
9
9
12.1768
9
85.8031
1
[-][-][-][-][-][-][-][+][-]
83.4507
[-][-][-][-][-][+][+][+][-]
-68.0847
3
-68.0847
9
2
[-][-][-][-][-][-][+][+][-]
[-][-][-][-][-][-][+][+][-]
67.4387
9
cyclic_double_queue is empty!
[-][-][-][-][-][-][-][-][-]
9
cyclic_double_queue is empty!
cyclic_double_queue is empty!
9
cyclic_double_queue is empty!
cyclic_double_queue is empty!
1
-40.9117
cyclic_double_queue is empty!
cyclic_double_queue is empty!
cyclic_double_queue is empty!
cyclic_double_queue is empty!
[-][-][-][-][-][-][-][+][-]
[-][-][-][-][-][-][+][+][-]
[-][-][-][-][-][-][+][+][-]
-95.4351
-54.2287
15.0547
28.3633
-66.1259
4
9
